pkg/gitreceive/build.go

```diff
@@ -256,8 +263,9 @@ func build(conf *Config, s3Client *s3.S3, builderKey, gitSha string) error {
 	if !usingDockerfile {
 		buildHook.Dockerfile = ""
 	} else {
+		json.Unmarshal([]byte("{}"), &procType)
```

Comment:
Why not just initialize a bare Procfile instead of unmarshalling a string? Seems a little over-engineered, but I guess it would work anyways.